tests/middlewares/Guard.test.js

```diff
@@ -0,0 +1,40 @@
+import chai from 'chai';
```

Comment:
Remove this file please

Reply:
I removed the file at some point. However, my test coverage dropped. The file contains unit tests for the Guard middleware. Thank you.